raven/src/main/java/com/getsentry/raven/DefaultRavenFactory.java

```diff
@@ -341,6 +315,182 @@ protected Marshaller createMarshaller(Dsn dsn) {
     }
 
     /**
+     * Returns whether or not to wrap the underlying connection in an {@link AsyncConnection}.
+     *
+     * @param dsn Dsn passed in by the user.
+     * @return whether or not to wrap the underlying connection in an {@link AsyncConnection}.
+     */
+    protected boolean getAsync(Dsn dsn) {
```

Comment:
This method name is a little weird. Not sure what a better name would be, though. `getAsyncEnabled` or something?